packages/apollo-server/src/index.ts

```diff
@@ -31,9 +35,25 @@ export class ApolloServer<Context> extends ExpressServer {
 
     opts.app = express();
     super(opts);
+
+    opts.app.use('/.health-check', (req, res, next) => {
```

Comment:
This seems worth breaking out into a separate commit.  Both to unblock merge-ability of this PR on its own, but also to help identify the feature in the commit flow (which will translate directly to the CHANGELOG!).

Reply:
Thank you! Good reminder to have a changelog entry :tada: